### PR TITLE
DEM-10 Test Runner working with jenkinds

### PR DESCRIPTION
**Pull request by indytechcook**
Some changes to the test runner.  There is a inject issue with the create project test.  Not sure why yet.

### CODE REVIEW

**Comment by indytechcook** [comment]
@tladendo can you give this a look please

**Comment by indytechcook** [comment]
@vietnogi and @tladendo please give it a review

File: gulp.config.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ module.exports = function () {
     ],
     specHelpers: [client + 'test-helpers/*.js'],
     specs: [clientApp + '**/*.spec.js'],
-    serverIntegrationSpecs: [client + '/tests/server-integration/**/*.spec.js'],
+    serverIntegrationSpecs: [client + '/**/*.spec.js'],
```
Comment (vietnogi) [inline gulp.config.js]: Odd that this path is absolute while others are relative.

File: gulpfile.js [hunk 2]
```diff
@@ -155,6 +155,7 @@ gulp.task('templatecache', function () {
 gulp.task('inject', ['jade', 'scss', 'ng-constants'], function (done) {
   log('Wire up css into the html, after files are ready');
 
+  //startTests(true /*singleRun*/, done);
```
Comment (vietnogi) [inline gulpfile.js]: Commented out?

**Comment by vietnogi** [comment]
Minor comments, otherwise, lgtm

File: gulp.config.js (hunk 1 above)
Comment (indytechcook) [inline gulp.config.js]: This isn't event used here.  It was for excluding files.  I'll remove it.

File: gulpfile.js (hunk 2 above)
Comment (indytechcook) [inline gulpfile.js]: Yeah.  Tests are run with gulp test

**Comment by tladendo** [comment]
lgtm

**Pull request merged**